packages/usuario/src/domains/usuarios.js

```diff
@@ -0,0 +1,35 @@
+const { Op } = require('sequelize');
+
+const findQuery = (usuarioModel) => async () => {
+  const rta = await usuarioModel.findAll({
+    where: {
+      deletedAt: {
+        [Op.is]: null,
+      },
+    },
+    /* include: [
+      {
+        association: 'customer',
+        include: ['user'],
+      },
+    ], */
+  });
+
+  return rta;
+};
+
+const findOneQuery = (usuarioModel) => async (id) => {
+  const rta = await usuarioModel.findByPk(id);
+  return rta;
+};
+
+const createQuery = (usuarioModel) => async (data) => {
+  const newUsuario = await usuarioModel.create(data);
+  return newUsuario;
+};
+
+module.exports = {
+  findQuery,
+  findOneQuery,
+  createQuery,
+};
```

Comment:
I think we should separet both methods in two files and export them in index.js from this directory
Could we add a new line here? You can read why it is needed [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)